packages/ListBox/src/components/Content/Content.js

```diff
@@ -15,17 +18,15 @@ const handleBlur = (state, dispatch, onCancelFooter) => () => {
   // via document.activeElement instead of returning
   // the body element automatically
   window.requestAnimationFrame(() => {
+    const activeElement = getActiveElement();
+
     // in case the on blur happens on the trigger it should ignore it and let the trigger to control the flow of the
     // popover
-    if (state.refTriggerContainer.current && state.refTriggerContainer.current.contains(document.activeElement)) {
+    if (state.refTriggerContainer.current && state.refTriggerContainer.current.contains(activeElement)) {
       return;
     }
 
-    if (
-      refListBoxContainer &&
-      refListBoxContainer.current &&
-      !refListBoxContainer.current.contains(document.activeElement)
-    ) {
+    if (refListBoxContainer && refListBoxContainer.current && !refListBoxContainer.current.contains(activeElement)) {
```

Comment:
I think `refListBoxContainer &&` is redundant 🤔

Reply:
You're right, a user-triggered blur even should not be executed before the DOM ref is set.  But I also don't mind this defensive practice, though `refListBoxContainer?.current?.contains()` would be much more succinct.  

Still, my intention here isn't really to refactor that, it's just to replace `document.activeElement` with the new `getActiveElement()` helper.